addons/account/migrations/12.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+column_copies = {
+    'account_payment_term_line': [
+        ('option', None, None),
+    ],
+}
+
+_column_renames = {
+    'account_tax': [
+        ('tax_adjustment', None),
+    ],
+    'account_tax_template': [
+        ('tax_adjustment', None),
+        ('cash_basis_account', 'cash_basis_account_id'),
+    ],
+    'account_chart_template': [
+        ('company_id', None),
+    ],
+    'account_invoice': [
+        ('reference_type', None),
+    ],
+    'account_invoice_line': [
+        ('layout_category_id', None),
+    ],
+}
+
+_table_renames = [
+    ('stock_incoterms', 'account_incoterms')
+]
+
+_model_renames = [
+    ('stock.incoterms', 'account.incoterms')
+]
```

Comment:
The stock.incoterms is introduced in the stock module in v11 https://github.com/odoo/odoo/blob/11.0/addons/stock/models/stock_incoterms.py
and in v11, account doesn depend on stock. So I am afraid, if you have only the account module and not the stock module installed, then I think this _table_renames and _model_renames will fail.

Reply:
put in base